src/components/App.js

```diff
@@ -0,0 +1,55 @@
+import Component from './Component.js';
+import Header from './Header.js';
+import TodoList from './TodoList.js';
+import todos from '../data/todos.js';
+import AddTodo from './AddTodo.js';
+
+class App extends Component {
+
+    render() {
+        const dom = this.renderDOM();
+
+        const header = new Header();
+        const headerDOM = header.render();
+
+        const todoList = new TodoList({ 
+            todos,
+            onRemove:(TodoToRemove) => {
+                const index = todos.indexOf(TodoToRemove);
+
+                todos.splice(index, 1);
+                todoList.update({ todos });
+            }
+        });
+
+        const todoListDOM = todoList.render();
+
+        const addTodo = new AddTodo({
+            onAdd: newTodo => {
+                todoList.unshift(newTodo);
```

Comment:
After you are able to submit - you need to rethink line 29. Where is todoList coming from? Are you sure you don't want to unshift (add to the beginning of the list) the todos that you're importing from your data? unshift is an array method but it looks like you're attempting to use it on an instance of the TodoList class.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift